code/modules/mob/living/silicon/robot/inventory.dm

```diff
@@ -38,6 +32,13 @@
 	else if(held_items[3] == O)
 		inv3.icon_state = "inv3"
 		held_items[3] = null
+
+	O.forceMove(module) //Return item to module so it appears in its contents, so it can be taken out again.
+
+	if(O.flags_1 & DROPDEL_1)
+		O.flags_1 &= ~DROPDEL_1 //we shouldn't HAVE things with DROPDEL_1 in our modules, but better safe than runtiming horribly
+
+	O.dropped(src)

```

Comment:
`forceMove()` calls `dropped(M)` (via `doMove`) if the item's loc is a mob.

Reply:
Are you sure? I can't find such a call.

Reply:
Nvm, found it